### PR TITLE
adding the MRP build doc

### PR DESCRIPTION
**Pull request by cicorias**
Missed this file in last PR

### CODE REVIEW

File: docs/Build-Install-MRP-App.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
+#Build and Install MRP App 
+This document explains how to build and manually install the MRP Application
+
+To be able to run through this walkthrough there is a dependency on Java 8 SDK and JRE8. These can be found at the link below:
+http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html
```
Comment (dtzar) [inline docs/Build-Install-MRP-App.md]: Missing dependencies for Gradle

File: docs/Build-Install-MRP-App.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+##Build the MRP Solution 
+In order to build the MRP solution we will need to install some more dependencies. 
+1.  To build the project, navigate to the OrderService folder with cmd/PowerShell console and initiate the build by calling the gradlew file with the parameter of the build.gradle file.
+
+`PS> .\gradlew build`
+
+Once run you can find the compiled jar file under the “build/libs” folder.
+
+##Open the solution in Eclipse
```
Comment (dtzar) [inline docs/Build-Install-MRP-App.md]: I technically don't need Eclipse.  Why not remove this and just go with gradle commands?

File: docs/Build-Install-MRP-App.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+
+`PS> .\gradlew build`
+
+Once run you can find the compiled jar file under the “build/libs” folder.
+
+##Open the solution in Eclipse 
+
+We recommend Eclipse IDE but you can use whatever IDE you want.
+
+1. To be able to add the OrderService to Eclipse as a project first we need to add the plugin.
```
Comment (dtzar) [inline docs/Build-Install-MRP-App.md]: This is only building the orderservice.  What If I want to build the integration or web service??

File: docs/Build-Install-MRP-App.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+2.  Open the file build.gradle.
+3.  Add the following line into the section of plugins.
+
+`apply pulgin: 'eclipse'`
+
+4. Run the following command in the PowerShell console
```
Comment (dtzar) [inline docs/Build-Install-MRP-App.md]: What if I am on an Linux box that has Gradle installed?

File: docs/Build-Install-MRP-App.md [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
+
+`PS:>.\gradlew eclipse
+PS:>.\gradlew buildDependents`
+
+5.  Open Eclipse and create a new workspace.
+6.  Right click on the Project Explorer and select import a new project.
+7.  Browse to the OrderService root folder and select it
+8.  Check the OrderService project and click finish. 
+
+##Create a Linux VM
```
Comment (dtzar) [inline docs/Build-Install-MRP-App.md]: uh?  What if I already am on a Linux machine?  Why do I need to create a VM now?  This is trying to host the app not build it.

File: docs/Build-Install-MRP-App.md [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+
+5.  Open Eclipse and create a new workspace.
+6.  Right click on the Project Explorer and select import a new project.
+7.  Browse to the OrderService root folder and select it
+8.  Check the OrderService project and click finish. 
+
+##Create a Linux VM 
+Create a Linux VM in Azure and signin onto it.
+
+##Install Dependencies
```
Comment (dtzar) [inline docs/Build-Install-MRP-App.md]: I had to install dependencies above.  Now again?   Merge into a section up top.

File: docs/Build-Install-MRP-App.md [hunk 7]
```diff
@@ -0,0 +1,205 @@
...
+`export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+export PATH=$PATH:/usr/lib/jvm/java-8-openjdk-amd64/bin`
+
+4.  To check your Java 8 installation, type “java -version”
+NOTE: if you are still referencing an older version of Java that was installed as part of your machine you can update this using the following command:
+
+`sudo update-alternatives --config java`
+You will be presented with a list of Java installations. Type the number you wish to set as your java default.
+
+##Setup MONGODB
```
Comment (dtzar) [inline docs/Build-Install-MRP-App.md]: Setup Mongo and Setup webservice are to try and host/run your app, not create the builds.  Do we have these steps already defined in our existing deployment scripts/docs?  This doc should be purely spit out builds for web, order, integration services.  Then link to the deployment doc(s) which will spin up the Linux VM, install dependencies, etc.

File: docs/Build-Install-MRP-App.md [hunk 8]
```diff
@@ -0,0 +1,205 @@
...
+
+5. Now there is data in the database you can check it with the following command 
+`> show dbs`
+
+6.  Copy the commands from Appendix A to insert sample data into the database 
+
+##Setup WEBSERVICE
+
+
+##APPENDIX A
```
Comment (dtzar) [inline docs/Build-Install-MRP-App.md]: None of this is needed in the doc because we have the Mongo records file.  You can just reference that.

**Comment by cicorias** [comment]
working on the updated guide that includes Linux steps as well..

File: docs/Build-Install-MRP-App.md (hunk 1 above)
Comment (cicorias) [inline docs/Build-Install-MRP-App.md]: Gradlew installs the gradle components by itself

File: docs/Build-Install-MRP-App.md (hunk 2 above)
Comment (cicorias) [inline docs/Build-Install-MRP-App.md]: removed

File: docs/Build-Install-MRP-App.md (hunk 4 above)
Comment (cicorias) [inline docs/Build-Install-MRP-App.md]: doesn't matter, the gradlew will pull it down as required and use.

File: docs/Build-Install-MRP-App.md (hunk 5 above)
Comment (cicorias) [inline docs/Build-Install-MRP-App.md]: new version will have Ubuntu only way.

**Pull request closed without merging**